app/models/employee.rb

```diff
@@ -0,0 +1,5 @@
+class Employee < ApplicationRecord
+  validates :name, presence: true
+  validates :redmine_user_id, presence: true
+  validates :open_investment_time, numericality: { less_than_or_equal_to: 80 }
```

Comment:
Would it make sense to extract this number into a configuration since you're using it often?

You've some different options for doing that:

* configuration files
* environment variables
* rails initializers
* a configuration class
* a configuration hash which you carry through the function calls and object initializations
* maybe you can come up with more...